msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/HttpHelper.java

```diff
@@ -105,6 +105,22 @@ IHttpResponse executeHttpRequest(HttpRequest httpRequest,
         return httpResponse;
     }
 
+    IHttpResponse executeHttpRequest(HttpRequest httpRequest) {
+        IHttpResponse httpResponse;
+
+        try {
+            httpResponse = executeHttpRequestWithRetries(httpRequest, httpClient);
+        } catch (Exception e) {
+            throw new MsalClientException(e);
```

Comment:
Is it really MsalClientException? Service errors should be MsalServiceException?

Reply:
Do you have tests covering this?

Reply:
This is the same exception type as the other versions of the `executeHttpRequest`, and after a quick look through the file history it's been that exception type since 2019.

It might be worth looking into exception types as an SDK consistency item, but to keep consistent with existing MSAL and avoiding breaking changes I'm leaving this as `MsalClientException`